src/aioswitcher/api/__init__.py

```diff
@@ -552,11 +612,21 @@ class SwitcherType2Api(SwitcherApi):
     """
 
     def __init__(
-        self, device_type: DeviceType, ip_address: str, device_id: str, device_key: str
+        self,
+        device_type: DeviceType,
+        ip_address: str,
+        device_id: str,
+        device_key: str,
+        token: str,
```

Comment:
You mentioned that there is no plan to break any older devices, There Switcher type 1 devices are very old, I don't understand why adding a token here

Reply:
@thecode I didn't put that on the SwitcherType1Api.. It's only for SwitcherType2Api.